src/abstractions/handler-base.ts

```diff
@@ -212,4 +212,43 @@ export abstract class HandlerBase<
 
         return restProps;
     }
+
+    //#region Public API
+    /**
+     * Get a boolean if dropdown is open or not.

```

Comment:
Maybe: 
`Get a boolean whether dropdown is open or not.`

Reply:
`Is dropdown open.`